docs/Maintain/build-and-start-a-node/03-ssl-certs-for-node.md

```diff
@@ -15,17 +15,22 @@ If you need an SSL certificate for your node, follow the steps below:
 
    ```bash
    acme.sh --issue --standalone -d your_node.example.com \
-   --renew-hook "cd /opt/your_node; ./stop.sh; ./start.sh"
+   --renew-hook "docker restart tpnode" <!-- Use this if you run with docker -->
+   --renew-hook "./bin/thepower stop && ./bin/thepower foreground" <!-- Use this if you run from source code -->
    ```
+   
+   :::info
```

Comment:
Add the empty lines
:::info

Use only one `--renew-hook`, depending on the run option (from docker / from source code)

:::

Reply:
Please, make some minor changes